### PR TITLE
style: reduce certificate's size

### LINKED ISSUE #104: Huawei's logo should be more clear
image

Hi, we noticed that this picture is a little blurred 🤣 We should find and use a more clear image.

https://www.apiseven.com/zh

https://api7ai.slack.com/archives/C01F08J8MS6/p1623663959067200?thread_ts=1623120990.032000&cid=C01F08J8MS6

### PR DESCRIPTION
**Pull request by juzhiyuan**
fix #104

### CODE REVIEW

**Comment by moonming** [comment]
Can the picture be smaller？

**Review by liuxiran — COMMENTED** [review]
Since we have already got the authentication, could we apply for a svg  from HUAWEI ?

**Comment by juzhiyuan** [comment]
done

File: views/Home/components/HomeCertificates/index.tsx [hunk 1]
```diff
@@ -29,48 +39,94 @@ const App: NextPage<Props, any> = ({ t }) => {
...
+            direction={{ base: "column", md: "row" }}
+            justifyContent="space-around"
+          >
+            <Box
+              textAlign="center"
+              as="a"
+              href="/zh/cert/huawei-stack-8.0"
+              target="_blank"
+            >
+              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px" />
```
Comment (liuxiran) [inline views/Home/components/HomeCertificates/index.tsx:59]: center the image would be better
image

```suggestion
              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px"  margin="0 auto"/>
```

File: views/Home/components/HomeCertificates/index.tsx [hunk 2]
```diff
@@ -29,48 +39,94 @@ const App: NextPage<Props, any> = ({ t }) => {
...
             <br />
-            <Box textAlign="center" as="a" href="/zh/cert/huawei-kunpeng" target="_blank">
-              <Image src="https://api7-website-1301662268.file.myqcloud.com/logo/logo-huawei-compatible2.png" maxH="130px" />
+            <Box
+              textAlign="center"
+              as="a"
+              href="/zh/cert/huawei-kunpeng"
+              target="_blank"
+            >
+              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px" />
```
Comment (liuxiran) [inline views/Home/components/HomeCertificates/index.tsx:70]:
```suggestion
              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px" margin="0 auto" />
```

**Review by liuxiran — APPROVED** [review]

**Pull request merged**